lib/pages/user_profile.dart

```diff
@@ -0,0 +1,185 @@
+import 'package:flutter/material.dart';
+import 'package:template_name/shared/behaviours/custom_scroll_behavior.dart';
+import 'package:template_name/shared/enums/page.dart';
+import 'package:template_name/shared/page_resolvers/navigator.dart';
+import 'package:template_name/shared/page_resolvers/positioning.dart';
+import 'package:template_name/shared/page_resolvers/page_resolver.dart';
+import 'package:template_name/shared/ui/cards/recipe_card.dart';
+import 'package:template_name/shared/ui/recipes/recipes_manager.dart';
+import 'package:template_name/shared/colors/default_colors.dart';
+import 'package:template_name/shared/ui/constants/custom_bottom_navigation_bar.dart';
+import 'package:template_name/shared/ui/recipes/recipes_manager.dart';
+import 'package:template_name/shared/ui/recipes/fav_recipes.dart';
+
+class UserProfile extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) => wrapWithScrollingView(Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: <Widget>[
+            switchPage(
+                context,
+                UpperSection(),
+                Page.RecipePage),
+            addPadding(
+                Text(
+                  'Favourite recipes',
+                  style: TextStyle(fontSize: 20.0, color: Colors.white),
+                ),
+                left: 16.0,
+                top: 8.0,
+                bottom: 8.0),
+            Expanded(
+              child: FavRecipesManager(),
+            )
+          ]));
+}  
+
+class UpperSection extends StatelessWidget {
```

Comment:
The New States should be in anew file to no longer existing files.

Reply:
refactored, no longer in the code